### PR TITLE
836 - New plugin to fix bug on Scan screen

### PR DESCRIPTION
**Pull request by jbdtky**
- [x] Start the camera when you come back from the settings on Android & iOS
- [x] Turn off the camera on Android
- [x] Review & test on iOS
- [x] Implement desktop

### CODE REVIEW

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        if let _ = message  {
+            pluginResult = CDVPluginResult(status: status, messageAs: message)
+        } else {
+            pluginResult = CDVPluginResult(status: status)
+        }
+        
+        commandDelegate.send(pluginResult, callbackId: callbackId)
+    }
+    
+    func onSuccess(_ result: String) {
```
Comment (brendoncoin) [inline plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: This function should be `fileprivate` like the others. How about making the whole extension fileprivate instead?

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        guard let readingCommand = self.readingCommand else {
+            return
+        }
+        
+        callback(readingCommand, status: CDVCommandStatus_OK, message: result)
+        self.readingCommand = nil
+    }
+    
+    func onFailed(_ error: QRReaderError) {
```
Comment (brendoncoin) [inline plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: This function should be `fileprivate` like the others. How about making the whole extension fileprivate instead?

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift (hunk 1 above)
Comment (jbdtky) [reply plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: Yes, sure, go ahead and fix it.

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift (hunk 2 above)
Comment (jbdtky) [reply plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: Yes, sure, go ahead and fix it.

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift (hunk 1 above)
Comment (jbdtky) [reply plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: Move the a private repo, will be fixed there.

File: plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift (hunk 2 above)
Comment (jbdtky) [reply plugins-bitcoincom/cordova-plugin-qrreader/src/ios/QRReader.swift]: Move to a private repo, will be fixed there.

File: src/js/routes.js [hunk 3]
```diff
@@ -1430,6 +1430,42 @@ angular.module('copayApp').config(function(historicLogProvider, $provide, $logPr
         channel = "firebase";
       }
       
+      //console.log('calling getInfo()...');
```
Comment (holemcross) [inline src/js/routes.js]: Extra comments. Should be removed.

**Pull request merged**